main.go

```diff
@@ -281,7 +282,9 @@ func generate(in configInput) ([]byte, error) {
 
 		if in.includeDirectPathAuthority {
 			c.Authorities[c2pAuthority] = Authority{
-				XdsServers:                         generateServerConfigsFromInputs("dns:///directpath-pa.googleapis.com", in),
+				// In the case of DirectPath, it is safe to assume that the operator is notified of missing resources.
+				// In other words, "ignore_resource_deletion" server_features is always set.
+				XdsServers:                         generateServerConfig(c2pTdURI, in.includeV3Features, true),
```

Comment:
@easwars, how would you feel about avoiding `generateServerConfigsFromInputs()` here and just hard-coding the config. I feel like using the function serves no purpose for DP. v3 should be hard-coded for it, and no conditionals at all. I feel like it got this way by trying too hard to reuse code.

Reply:
I agree. It seems like we have to go back and forth. and the helper is only used in two places. but for different reasons.
Done.